auth4genai/snippets/how-tos/github/langgraph.mdx

```diff
@@ -39,12 +49,12 @@ import { z } from "zod";
 export const listRepositories = withGitHub(
   tool(async () => {
     // Get the access token from Auth0 AI
-    const accessToken = getAccessTokenForConnection();
+    const credentials = getCredentialsForConnection();
```

Comment:
Missing `import` statement for `getCredentialsForConnection()` ☝️ but from the start, why was this method call changed?